### DISCUSSION TITLE
Multi-language

### DISCUSSION THREAD

**Discussion by ClaudineiMonteiro** [balta-io/forum#122] [Tira Dúvidas] [answered]
Boa tarde galera,
Imaginando que está aplicação fosse mult-language, o que seria mais indicado:
1) Criar os resources de idioma no projeto de dominio, devido as mensagens de validação depender do idioma.
Ou
2) Criar um projeto de idiomas e referenciar no dominio. Lembrando que nesse projeto de idioma teria apenas os resources e classes e interfaces necessárias para a tradução. Só código C# para não poluir o dominio.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @ClaudineiMonteiro, ótima pergunta...
Inicialmente, não vejo problema em ter os resources no domínio, visto que as mensagens são concentradas lá e técnicamente todos os outros projetos podem referenciar o domínio.

Agora se estiver trabalhando com múltiplos domínios e quiser compartilhar a mesma mensagem globalizada entre eles, aí a saída seria deixar em um projeto a parte ou no Shared.

**Reply by ClaudineiMonteiro** [reply]
Muito obrigado pela resposta, vocês estão fazendo um ótimo trabalho estou muito satisfeito com minha assinatura.
